src/components/editor/metaform-editor-right-drawer-feature.tsx

```diff
@@ -567,6 +570,9 @@ const MetaformEditorRightDrawerFeature: FC<Props> = ({
           value={ selectedMemberGroupId }
           onChange={ event => handleMemberGroupChange(event.target.value) }
         >
+          <MenuItem value={ NOT_SELECTED } key={uuid4()}>
+            { strings.draftEditorScreen.editor.memberGroups.noMemberGroupSelected }
+          </MenuItem>
```

Comment:
What's the point of this, why would user want to select an empty group?

Reply:
Used to delete member groups from only 1 option.
Currently when u switch member group switch off, it deletes member groups from all options.

Reply:
Users can just select no permission from the permission select them the permission for that group is disabled

Reply:
We though its little more clearly when there is not selected option also in member groups.